src/main/java/com/amazon/aocagent/enums/GenericConstants.java

```diff
@@ -49,6 +49,26 @@
   CANDIDATE_PACK_TO("build/candidate.tar.gz"),
   CANDIDATE_DOWNLOAD_TO("build/candidate-downloaded.tar.gz"),
   CANDIDATE_UNPACK_TO("."),
+
+  //ECS
+  LAUNCH_TYPE("launchType"),
+  DEPLOY_MODE("deployMode"),
+  ECS_IAM_ROLE_NAME("ecsInstanceRole"),
+  ECS_EC2_AMI_ID("ami-004e1655142a7ea0d"), // us-west-2
+  EC2_INSTANCE_ECS_TAG_VAL("aoc-integ-test-ecs"),
+  ECS_SIDECAR_CLUSTER("aoc-sidecar-integ-test"),
+  ECS_EC2_INSTANCE_ID("aoc-sidecar-ec2"),
+  ECS_FARGATE_INSTANCE_ID("aoc-sidecar-fargate"),
+  // To be edited under prod test account
+  ECS_TASK_ROLE("arn:aws:iam::252610625673:role/CWAgentECSTaskRole"),
```

Comment:
we can't put account id here as this integ-test framework is meant to be used for any account.